README.md

```diff
@@ -87,8 +88,52 @@ Google has partnered with [Disconnect.me](https://Disconnect.me), a prominent in
 #### Publication of the Masked Domain List
 The MDL ([initial version](https://github.com/GoogleChrome/ip-protection/blob/main/Masked-Domain-List.md)) will be hosted on GitHub. Periodically, domains may be added or removed based on the fingerprinting detection system and updates to Disconnect's published list. Chrome will also remove domains that have successfully obtained an appeal. The published MDL will be the latest version used by Chrome.
 
+For general inquiries about the MDL, such as those regarding domain inclusions, exclusions or
+updates, please contact <a href="mailto:mdl_inquiries@disconnect.me">mdl_inquiries@disconnect.me</a>.
+Please note that response times for general inquiries may not adhere to the
+<a href="#policies-and-timelines">timelines listed for appeals</a>.
+
 #### Appeals
-We recognize the importance of implementing an appeals process for our list-based approach. Appeals permit companies to make a claim that their domain on the MDL does not meet the inclusion criteria and ought to be removed, thereby allowing that domain to continue to receive users' original IP addresses in a third-party context in Incognito. Before launching IP Protection, we will establish the appeals process to ensure companies have adequate opportunity to seek an appeal and receive a decision. The appeals process is being designed to align with governance principles for Privacy Sandbox under discussion with the UK’s Competition and Markets Authority ([see CMA's 2024 Q3 report](https://www.gov.uk/cma-cases/investigation-into-googles-privacy-sandbox-browser-changes#q2q3-2024)).
+We recognize the importance of implementing an appeals process for our list-based approach. Appeals
+permit companies to make a claim that their domain on the MDL does not meet the inclusion criteria
+and ought to be removed, thereby allowing that domain to continue to receive users' original IP
+addresses in a third-party context in Incognito. Before launching IP Protection, we will establish
+the appeals process to ensure companies have adequate opportunity to seek an appeal and receive a
+decision.
+
+The appeals process will be available starting **April 15th, 2025** to provide domain owners
```

Comment:
This section appears to be written in the future tense. But since that date is today, we can say that the process is now live and describe it in the present tense throughout this section